ingest/rules/nextclade.smk

```diff
@@ -123,40 +129,109 @@ rule aggregate_gene_coverage_by_gene:
     Aggregate the gene coverage results by gene
     """
     input:
-        gene_coverage=expand("data/translations/{serotype}/{{gene}}/gene_coverage.tsv", serotype=SUPPORTED_NEXTCLADE_SEROTYPES),
+        gene_coverage=expand("data/v-gen-lab/{serotype}/translations/{{gene}}/gene_coverage.tsv", serotype=SUPPORTED_NEXTCLADE_SEROTYPES),
     output:
         gene_coverage_all="results/{gene}/gene_coverage_all.tsv",
+    log:
+        "logs/v-gen-lab/{gene}/aggregate_gene_coverage_by_gene.txt",
+    benchmark:
+        "benchmarks/v-gen-lab/{gene}/aggregate_gene_coverage_by_gene.txt",
     shell:
         """
         tsv-append -H {input.gene_coverage} > {output.gene_coverage_all}
         """
 
-rule append_gene_coverage_columns:
+rule combine_gene_coverage_columns:
     """
     Append the gene coverage results to the metadata
+    Since gene coverage values should be a value between 0 and 1, empty fields should be filled with 0's
     """
     input:
-        metadata="data/metadata_nextclade.tsv",
+        metadata="data/metadata_all.tsv",
         gene_coverage=expand("results/{gene}/gene_coverage_all.tsv", gene=config["nextclade"]["gene"])
     output:
-        metadata_all="results/metadata_all.tsv",
+        gene_coverage_combined="results/gene_coverage_combined.tsv",
     params:
         id_field=config["curate"]["output_id_field"],
+    log:
+        "logs/v-gen-lab/combine_gene_coverage_columns.txt",
+    benchmark:
+        "benchmarks/v-gen-lab/combine_gene_coverage_columns.txt",
     shell:
         """
-        cp {input.metadata} {output.metadata_all}
+        tsv-select -H -f "{params.id_field}" {input.metadata} > {output.gene_coverage_combined}
         for FILE in {input.gene_coverage}; do
             tsv-join -H \
                 --filter-file $FILE \
                 --key-fields {params.id_field} \
                 --append-fields '*_coverage' \
                 --write-all 0 \
-                {output.metadata_all} \
+                {output.gene_coverage_combined} \
             > results/temp_aggregate_gene_coverage.tsv
-            mv results/temp_aggregate_gene_coverage.tsv {output.metadata_all}
+            mv results/temp_aggregate_gene_coverage.tsv {output.gene_coverage_combined}
         done
         """
 
+rule append_nextclade_columns:
+    """
+    Append the nextclade results to the metadata
+    """
+    input:
+        metadata="data/metadata_all.tsv",
+        genotype_nextclade="results/v-gen-lab/nextclade_metadata.tsv",
+    output:
+        metadata="data/metadata_nextclade.tsv",
+    params:
+        output_nextclade_fields=",".join([f'{value}' for key, value in config["nextclade"]["field_map"].items()][1:]),
```

Comment:
The `output_nextclade_fields` doesn't look like it's used in the `augur merge` command

Reply:
Agree, unused parameters dropped as part of https://github.com/nextstrain/dengue/pull/93/commits/c0af2c770c150a3467a1b15ceab15acfe71914c0